src/ignitus-Routes/publicRoutes.js

```diff
@@ -75,6 +76,7 @@ export const PublicRoutes = props => {
         />
         <Route path="/Signup" component={commonSignUpUI} />
         <Route path="/Login" component={commonLoginUI} />
+        <Route path="/jobConfirmation" component={JobConfirmation.components.JobConfirmation} />
```

Comment:
As per the path it should work, Am I missing something 🤷‍♂️ 😅.

Reply:
Yes its working fine. I didn't say it was not working.

Reply:
Yup but what exactly is the route I should navigate to, to see your changes.
this doesn't seem to work `https://deploy-preview-589--peaceful-nightingale-55a1b7.netlify.com/jobDescription`

Reply:
Sorry my bad 😅 ignore.

Reply:
The code automatically deploys on netlify , even though the PR hasn't been merged?